cypress/support/index.js

```diff
@@ -37,18 +37,33 @@ Cypress.Commands.add("waitForGraph", operationName => {
       }
     }
   });
-  cy.wait(`@${operationName}`);
 });
 
-Cypress.Commands.add("sendRequestWithQuery", query =>
-  cy.request({
-    body: {
-      method: "POST",
+Cypress.Commands.add("sendRequestWithQuery", query => {
+  const body = {
+    method: "POST",
+    query,
+    url: urlList.apiUri
+  };
+  return cy.sendRequest(body, "auth");
+});
+
+Cypress.Commands.add(
```

Comment:
Do we need separate function to that? Shouldnt sendRequestWithQuery be reused, just with different token if needed?